src/reconcile.py

```diff
@@ -116,23 +138,58 @@ def _local_and_server_dir_local_group_server(
                     title=path_info.navlink_title, link=table_row.navlink.link, hidden=False
                 ),
             ),
-            content_change=None,
+        ),
+    )
+
+
+def _local_and_server_external_ref_local_external_ref_server(
+    item_info: types_.IndexContentsListItem, table_row: types_.TableRow
+) -> tuple[types_.UpdateAction | types_.NoopAction, ...]:
+    """Handle the case where the item is an external reference locally and on the server.
+
+    Args:
+        item_info: Information about the local contents item.
+        table_row: A row from the navigation table.
+
+    Returns:
+        The action to execute against the server.
+
+    """
+    if (
+        table_row.navlink.title == item_info.reference_title
+        and table_row.navlink.link == item_info.reference_value
+    ):
+        return (
+            types_.NoopExternalRefAction(
+                level=item_info.hierarchy, path=item_info.table_path, navlink=table_row.navlink
+            ),
+        )
+    return (
+        types_.UpdateExternalRefAction(
+            level=item_info.hierarchy,
+            path=item_info.table_path,
+            navlink_change=types_.NavlinkChange(
+                old=table_row.navlink,
+                new=types_.Navlink(
+                    title=item_info.reference_title, link=item_info.reference_value, hidden=False
```

Comment:
Yes that is probably better, there isn't really a use case for hidden external items. It is probably better for consistency and so that the behaviour is not unexpected for users

Reply:
Fixed now